src/index.js

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter as Router } from 'react-router-dom';
 import './index.css';
 import App from './App';
 import 'bootstrap/dist/css/bootstrap.css';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+ReactDOM.render(<Router><App /></Router>, document.getElementById('root'));
```

Comment:
Here's another formatting option for readability, where you separate each argument of a function onto its own line:
```javascript
 ReactDOM.render(
  <Router><App /></Router>, 
  document.getElementById('root')
); 
```
or, for even further readability, separating separate components onto their own lines:
```javascript
ReactDOM.render(
  <Router>
    <App />
  </Router>, 
  document.getElementById('root')
); 
```